pkg/clustertree/cluster-manager/controllers/pod/leaf_pod_controller.go

```diff
@@ -141,7 +143,9 @@ func (r *LeafPodReconciler) SetupWithManager(mgr manager.Manager) error {
 
 	return ctrl.NewControllerManagedBy(mgr).
 		Named(LeafPodControllerName).
-		WithOptions(controller.Options{}).
+		WithOptions(controller.Options{
+			MaxConcurrentReconciles: r.Options.MaxConcurrentReconciles,
```

Comment:
It is strange when the options are passed anywhere

Reply:
Replace it with an environment variable?

Reply:
can we conduct a global option？

Reply:
the ```MaxConcurrentReconciles``` is a global option